handbook/product/README.md

```diff
@@ -393,7 +393,11 @@ Directly Responsible Individuals (DRI) engage in the ritual(s) below at the freq
 | Bug review (CX)                               | Weekly (Tuesday)         | The product manager meets with the engineering manager and a product quality engineer to ensure synchronization on bug prioritization and actions preceding CX estimation sessions.                                                                                                         | Mo Zhu            |
 | Churned bug review                            | Sprintly (Wednesday)     | The Head of Product meets with the group product managers to examine churned bugs and categorize them as either continue, needs prioritization, or de-prioritize.                                                                                                                                     | Mo Zhu            |
 | Churned bug clean-up | Sprintly | Following the bug de-prioritization meeting, Product Operations completes the churned bug clean-up, ensuring all necessary follow-up tasks are actioned to classify bugs as continue, needs prioritization, or de-prioritized. This may include relocating bug tickets, adjusting labels, communicating with stakeholders, writing documentation, and closing issues. | Isabell Reedy |
+Bug round-up | Sprintly (Tuesday) | After each sprint, Product Operations will compile a list of churned bugs, which will detail the issue number, bug specifics, bug age, and an area for product managers to suggest subsequent actions.| Isabell Reedy |
+Quality check | Daily | Every day, Product Operations will review the "designed" column on the drafting board to ensure all product action items are complete. | Isabell Reedy |
```

Comment:
The part about the "action items are complete" is great. It makes me realize that we need a definition of complete. Let's do that separately, but maybe could you pick a reasonable place in the docs to link to (either an existing section or a new section that is blank with a TODO) here?

Reply:
@zhumo I'm going to submit a PR for definition of complete (based on the issue template) and then will link it within the ritual.